src/wallet/wallet.cpp

```diff
@@ -4315,6 +4316,76 @@ bool CWallet::BackupWallet(const std::string& strDest) const
     return GetDatabase().Backup(strDest);
 }
 
+bool CWallet::QueuedTransactionExists(const uint256& txid)
```

Comment:
This method can be marked as `const`, right?  If the mutex is the reason it is not, then either the mutex should IMHO be `mutable`, or the lock should be `AssertLockHeld` instead (since you acquire the wallet lock anyway in the RPC code as far as I've seen).